### ISSUE TITLE
we can improve accuracy by changing random_state to 42

### ISSUE THREAD

**Issue by kushal121231** [DARK-art108/Bank-Note-Authentication-End-to-End-Project-1#2]
if we change the random_state =42 in the train_test_split. we can improve  the accuracy to 0.995
